### PR TITLE
AG-515: Gene search error messages wrap and do not highlight

### PR DESCRIPTION
**Pull request by JessterB**
See AG-515 for the before and after screen shots. It looks much better to me with these changes, but I'm far from a CSS wizard...

### CODE REVIEW

File: src/app/genes/gene-search/gene-search.component.html [hunk 1]
```diff
@@ -24,11 +24,13 @@
             </div>
         </div>
         <ul *ngIf="errorMessage"  class="filter-select">
-            <li class="filter-select-list" [class.italic-result]="isNotFound">
+            <li class="filter-select-error" [class.italic-result]="isNotFound">
                 <div class="gene-name row">
-                    <div class="col-md-12 col-sm-12 col-12">
-                        {{errorMessage}}
-                    </div>
+                    <span class="gene-search-error">
```
Comment (jay-hodgson) [inline src/app/genes/gene-search/gene-search.component.html]: Bit dangerous putting this between the bootstrap grid definitions (row and col-*).  Does it still work if you put span inside the col div (a child of the <div class="col-md-12 col-sm-12 col-12" /> instead)?

File: src/app/genes/gene-search/gene-search.component.scss [hunk 2]
```diff
@@ -117,14 +117,15 @@
     margin-right: 30px;
 }
 
+.filter-select-error,
 .filter-select-list {
     cursor: pointer;
     padding: 5px 10px;
-    height: 60px;
+    height: auto;
```
Comment (jay-hodgson) [inline src/app/genes/gene-search/gene-search.component.scss]: What's the height if you do not set it to auto here?  Is it not auto?

**Review by jay-hodgson — APPROVED** [review]

File: src/app/genes/gene-search/gene-search.component.scss (hunk 2 above)
Comment (JessterB) [reply src/app/genes/gene-search/gene-search.component.scss]: It is! Good catch, I've removed the unnecessary height attribute.

File: src/app/genes/gene-search/gene-search.component.html (hunk 1 above)
Comment (JessterB) [reply src/app/genes/gene-search/gene-search.component.html]: I moved the span inside the div, and tweaked a few things to get it working.

**Pull request merged**